terraform/aws/variables.tf

```diff
@@ -733,3 +733,11 @@ variable "ibsm_project_tag" {
   type        = string
   default     = ""
 }
+
+# Variable defaults
+
+variable "destroy_timeout" {
```

Comment:
As terraform/aws/main.tf is only forward it to hana module, you can name it `hana_destroy_timeout`